pokemongo_bot/__init__.py

```diff
@@ -28,6 +28,22 @@
 
 
 class PokemonGoBot(object):
+
+    WORKERS = [
```

Comment:
I think it wasn't properly merged.

They just have removed `WORKERS` https://github.com/PokemonGoF/PokemonGo-Bot/commit/5eef040f2b31f1ab70d106f39165610c03f7542c